test_conformance/vulkan/test_vulkan_interop_buffer.cpp

```diff
@@ -288,7 +323,14 @@ int run_test_with_two_queue(cl_context &context, cl_command_queue &cmd_queue1,
 
```

Comment:
We need to wait on OpenCL to finish the work.  I suggest we flush both queues to avoid deadlock, then either finish or wait_for events.  For example:

`
if(use_fence)
{
    clFlush(cmd_queue1);
    clFlush(cmd_queue2);
    err = clWaitForEvents(...);
    }
}
`